### PR TITLE
fix(mailviewer): Load HTML view faster by pre-caching it

### PR DESCRIPTION
**Pull request by castaway**
Fixes (hopefully) #998

### CODE REVIEW

File: src/app/mailviewer/singlemailviewer.component.ts [hunk 1]
```diff
@@ -390,7 +374,9 @@ export class SingleMailViewerComponent implements OnInit, DoCheck, AfterViewInit
         this.folder = res.folder;
         // ProgressDialog.close();
         if (this.mailObj.seen_flag === 0) {
+          console.log('markSeen pre');
```
Comment (tadzik) [inline src/app/mailviewer/singlemailviewer.component.ts:377]: Leftovers :)

**Review by tadzik — CHANGES_REQUESTED** [review]

**Comment by tadzik** [comment]
Looks good apart of the leftover logs :)

**Review by gtandersen — APPROVED** [review]
Great, @castaway!

File: src/app/mailviewer/singlemailviewer.component.ts (hunk 1 above)
Comment (castaway) [reply src/app/mailviewer/singlemailviewer.component.ts]: fixed

**Review by tadzik — APPROVED** [review]

**Pull request merged**